WordPress/src/main/java/org/wordpress/android/ui/posts/EditPostActivity.java

```diff
@@ -404,6 +406,40 @@ public void run() {
         ActivityId.trackLastActivity(ActivityId.POST_EDITOR);
     }
 
+    // this method aims at recovering the current state of media items if they're inconsistent within the PostModel.
+    private void resetUploadingMediaToFailedIfNotInProgressOrQueued(Context context, PostModel post) {
```

Comment:
It seems unnecessary to pass the context and the post since the context is this activity and the post is `mPost`.